### PR TITLE
Add tex content and metadata for mizrahi2021re

### PR DESCRIPTION
**Pull request by dmizr**
Adds the tex content and the metadata for the paper "[Re] Can gradient clipping mitigate label noise?"

**Compilation issues:**
PDF does not compile well on my local setup (missing bibliography and bad layout on page 1), but works well on the Overleaf version of the ReScience template.
- Compilation log can be found at https://github.com/dmizr/RC-2020/blob/main/mizrahi2021re/journal/article.log
- Compiled PDF can be found at https://github.com/dmizr/RC-2020/blob/main/mizrahi2021re/journal/article.pdf

### CODE REVIEW

File: mizrahi2021re/openreview/packages.tex [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
+\usepackage{hyperref}       % hyperlinks
+\usepackage{url}            % simple URL typesetting
+\usepackage{booktabs}       % professional-quality tables
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+\usepackage[dvipsnames]{xcolor}
```
Comment (koustuvsinha) [inline mizrahi2021re/openreview/packages.tex:26]: From the logs, it seems this import is the issue. Can you try removing this package?

(https://github.com/dmizr/RC-2020/blob/main/mizrahi2021re/journal/article.log#L1360)

**Review by koustuvsinha — COMMENTED** [review]
Try removing package `xcolor` from your packages.tex

File: mizrahi2021re/openreview/packages.tex (hunk 1 above)
Comment (dmizr) [reply mizrahi2021re/openreview/packages.tex]: Thank you! This import was definitely problematic and the locally compiled version looks much better now.
However, there still seems to be a few issues remaining, such as the font not matching the one used in other ReScience articles.
- New compiled PDF can be found here: https://github.com/ReScience/RC-2020/blob/25d68c6c427676b32845527f9a269cee7cd9375b/mizrahi2021re/journal/article.pdf
- New log: https://github.com/ReScience/RC-2020/blob/25d68c6c427676b32845527f9a269cee7cd9375b/mizrahi2021re/journal/article.log

Comment (koustuvsinha) [reply mizrahi2021re/openreview/packages.tex]: I think this is good enough! We can inspect the font mismatch on our end!

File: mizrahi2021re/journal/metadata.yaml [hunk 2]
```diff
@@ -66,51 +54,42 @@ data:
 
 # Information about the original article that has been replicated
 replication:
- - cite: # Full textual citation
- - bib:  # Bibtex key (if any) in your bibliography file
- - url:  # URL to the PDF, try to link to a non-paywall version
+ - cite: "Aditya Krishna Menon, Ankit Singh Rawat, Sanjiv Kumar, and Sashank Reddi. Can gradient clipping mitigate label noise?. In International Conference on Learning Representations (ICLR).2020." # Full textual citation
+ - bib:  \cite{menon_can_2019} # Bibtex key (if any) in your bibliography file
```
Comment (koustuvsinha) [inline mizrahi2021re/journal/metadata.yaml:58]: Please remove the `\cite`, just provide the bibkey `menon_can_2019`

**Comment by dmizr** [comment]
I changed the bibtex key in the metadata file as requested.
I just have one question/request regarding citations for our paper. Would it be possible to have citations in brackets instead of in superscript? Many articles in ReScience use brackets for citations, which would be useful in our case as it would avoid mistaking them for footnotes.

File: mizrahi2021re/openreview/packages.tex [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+%     \usepackage[preprint]{neurips_2019}
+
+% to compile a camera-ready version, add the [final] option, e.g.:
+%\usepackage[final]{neurips_2019} % Remove for ReScience
+
+% to avoid loading the natbib package, add option nonatbib:
+% \usepackage[nonatbib, final]{neurips_2019}
+
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
```
Comment (koustuvsinha) [inline mizrahi2021re/openreview/packages.tex:19]: Please remove this import `fontenc` as this appears to break ReScience fonts (check #4 for discussions)

**Comment by koustuvsinha** [comment]
> Would it be possible to have citations in brackets instead of in superscript?

I think ReScience formatting prefers superscript if you cite by name (`\citet`). If you would like to have a separation, then I would suggest replacing the `\citet`s with `\cite`

**Comment by dmizr** [comment]
> Please remove this import `fontenc` as this appears to break ReScience fonts (check #4 for discussions)

Thank you! This fixes the font issues we were having.

In addition, I've found that switching from natbib citation commands (`\citep`, `\citet`) to biblatex commands (`\parencite`, `\textcite`) leads to citations in brackets instead of superscript. Is this the intended behavior? If not, I can easily switch back to `\citep` and `\citet` to keep the superscript citations.

The compiled PDF can be found at: https://github.com/ReScience/RC-2020/blob/bcedf2216ea7efa10eba32226624fc01d66de18c/mizrahi2021re/journal/article.pdf

**Comment by koustuvsinha** [comment]
I think that is great :) We are okay with any consistent citation style you choose.

**Comment by koustuvsinha** [comment]
Latex looks good to me. @sashavor please review the content once, before we proceed to generate the editorial information

**Comment by sunniesuhyoung** [comment]
Hi @dmizr ! Could you share your Overleaf project where you compiled your camera-ready version with the ReScience template? I'm having lots of trouble compiling the PDF on my local machine, so I want to try compiling through Overleaf. Thank you!

**Comment by dmizr** [comment]
> Hi @dmizr ! Could you share your Overleaf project where you compiled your camera-ready version with the ReScience template? I'm having lots of trouble compiling the PDF on my local machine, so I want to try compiling through Overleaf. Thank you!

@sunniesuhyoung 
Sure! The Overleaf version of my paper using the ReScience template can be found [here](https://www.overleaf.com/read/wsqjvfqzznpc). It was adapted from the [ReScience Overleaf template](https://www.overleaf.com/read/kfrwdmygjyqw).

**Pull request merged**